quirks/websites-with-shared-credential-backends.json

```diff
@@ -17,6 +76,7 @@
         "amazon.com.au",
         "amazon.com.br",
         "amazon.ca",
+        "amazon.cn",
```

Comment:
Are we sure that amazon.cn shares a credential backend with the rest?

Reply:
Huh, I actually can't log in on that website with my amazon.com credentials, so no. It does seem to be owned by Amazon, but I guess they didn't merge their database of user accounts.

My bad, I'll remove it. Thanks, good catch.

Reply:
amazon.co.jp was mistaken as well it seems